app/controllers/admin/contests_controller.rb

```diff
@@ -18,6 +18,7 @@ def show
   def new
     authorize :contests, :new?
 
+    @contest_grups = ContestGroup.all()
```

Comment:
The convention used all around the project is to omit parentheses when there are no arguments. Please remove them from the ruby code (`ContestGroup.all()` should be `ContestGroup.all`)